src/Debugger.php

```diff
@@ -98,7 +98,10 @@ private function isRequestIgnored(ServerRequestInterface $request): bool
                 return true;
             }
         }
-        return false;
+        return (bool) ($request->hasHeader('X-Debug-Ignore') && $request->getHeaderLine('X-Debug-Ignore') === 'true')
+
+
+         ;
```

Comment:
```suggestion
        return $request->getHeaderLine('X-Debug-Ignore') === 'true';
```

When header not exist `getHeaderLine()` returns empty string.

Reply:
Then the condition turns to `false`?

Reply:
That's what this needs